modules/api-custom-image-creating.adoc

```diff
@@ -77,8 +79,10 @@ The example YAML file includes the following information:
 <6> The version for the image. You can configure multiple versions for the same image. For this example, the version is 1.0.
 <7> An annotation that gives the user information about the Python packages and versions that are pre-installed in the image.
 <8> An annotation that specifies information such as the Python version, Jupyter version, or CUDA version.
-<9> An annotation that specifies whether the `ImageStream` version is the default version of the image. Set this field to `'true'` if the `ImageStream` version is the default. Otherwise, set it to `'false'`. You must specify the `opendatahub.io/workbench-image-recommended` annotation field if there are multiple versions of the image with different configurations. If you have only one version of the image, set the field to `'true'`.
-<10> The image registry URL to store the image stream. Store the image stream in an image registry outside of the cluster where {productname-short} is installed.
+<9> An annotation that specifies whether the `ImageStream` version is the default version of the image. Set this field to `'true'` if the `ImageStream` version is the default. Otherwise, set it to `'false'`. You must specify the `opendatahub.io/workbench-image-recommended` annotation field if there are multiple versions of the image with different configurations. If you have only one version of the image, set the field to `'true'`. 
+<10> An annotation that specifies whether the image version has tags that are outdated and out of the regular maintenance cycle.
+<11> An annotation that is a reference to the build commit with the ID for the updated information.
+<12> The image registry URL to store the image stream. Store the image stream in an image registry outside of the cluster where {productname-short} is installed.
```

Comment:
```suggestion
<12> The image registry path where the image has been uploaded.
```

> image registry URL

it's not URL because it does not (and cannot have) e.g. scheme://

> Store the image stream in an image registry outside of the cluster where {productname-short} is installed.

Store it wherever it works for you.

Reply:
It has to be
* Accessible to the open shift cluster. Important for disconnected clusters that cannot connect to arbitrary registry anywhere on the internet.
* If there is auth required to pull the image, the cluster has to have a pull secret set up so it can pull from the registry.

In general, I believe open shift admins are well aware of both considerations.